src/stylesheets/components/_banner.scss

```diff
@@ -277,6 +289,12 @@ $banner-icon-close: (
         @include u-pin("y");
         @include u-pin("right");
       }
+
+      @media (forced-colors: active) {
```

Comment:
We could move this media query inside the `:after` above. They should also be `::before` and `::after`.